client/asset/eth/multirpc.go

```diff
@@ -1763,15 +1713,15 @@ func domain(addr string) (string, error) {
 // requires by sending a series of requests and verifying the responses. If a
 // provider is found to be compliant, their domain name is added to a list and
 // stored in a file on disk so that future checks can be short-circuited.
-func checkProvidersCompliance(ctx context.Context, providers []*provider, log dex.Logger) error {
+func checkProvidersCompliance(ctx context.Context, providers []*provider, compat *CompatibilityData, log dex.Logger) error {
 	for _, p := range providers {
 		// Need to run API tests on this endpoint.
-		for _, t := range newCompatibilityTests(p.ec, p.chainID, p.net, log) {
+		for _, t := range newCompatibilityTests(p.ec, compat, log) {
 			ctx, cancel := context.WithTimeout(ctx, defaultRequestTimeout)
 			err := t.f(ctx, p)
 			cancel()
 			if err != nil {
-				return fmt.Errorf("RPC Provider @ %q has a non-compliant API: %v", p.host, err)
+				return fmt.Errorf("%s: RPC Provider @ %q has a non-compliant API: %v", t.name, p.host, err)
```

Comment:
I see this error log in the node after the compatibility test.. I guess the connection is not being closed properly after the tests. Probably nbd though.

<img width="1351" alt="Screenshot 2023-07-11 at 5 36 42 AM" src="https://github.com/decred/dcrdex/assets/6186350/00ab5ac3-0e7b-4b4f-a81d-13e62bc0773b">

Reply:
What were you running when you saw this message? dexc or a test?

Reply:
dexc